client/components/command-palette/use-command-filter.tsx

```diff
@@ -0,0 +1,20 @@
+export const useCommandFilter = () => {
+	const commandFilter = ( value: string, search: string ) => {
+		const [ beforeSeparator, afterSeparator ] = value.split( '||' );
+
+		// Check if the search matches the part before the separator
+		if ( beforeSeparator.includes( search ) ) {
+			return 1;
+		}
+
+		// Check if there is an afterSeparator and the search matches it
+		if ( afterSeparator && afterSeparator.includes( search ) ) {
+			return 0.5;
```

Comment:
@sejas - as mentioned on Slack, I think the problem is with this line - for some reason, it does not get ranked second. It does get filtered though. I am starting to think that since this function initially expects one string, it might  not work with something like `0.5`

Reply:
@sejas should we just return `1` for a match either in `label` or in `searchLabel` do you think or do you think we should leave as is for now in case the bug gets fixed?

Reply:
I will leave it as is and we can refine later!